packages/flutter/lib/src/material/drawer.dart

```diff
@@ -231,6 +237,7 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   @override
   void initState() {
     super.initState();
+    _color = ColorTween(begin: Colors.transparent, end: widget.scrimColor);
```

Comment:
I believe you still want to keep a default color for widget.scrimColor, since the tween expects both arguments to not be null.

Reply:
you are right, done!